--- ../Helper/Helper.php~
+++ ../Helper/Helper.php
@@ -47,9 +47,9 @@
     {
         $string ??= '';

-        if (preg_match('//u', $string)) {
-            return (new UnicodeString($string))->width(false);
-        }
+//        if (preg_match('//u', $string)) {
+//            return (new UnicodeString($string))->width(false);
+//        }

         if (false === $encoding = mb_detect_encoding($string, null, true)) {
             return \strlen($string);
@@ -66,9 +66,9 @@
     {
         $string ??= '';

-        if (preg_match('//u', $string)) {
-            return (new UnicodeString($string))->length();
-        }
+//        if (preg_match('//u', $string)) {
+//            return (new UnicodeString($string))->length();
+//        }

         if (false === $encoding = mb_detect_encoding($string, null, true)) {
             return \strlen($string);
